helpers.py

```diff
@@ -103,5 +103,14 @@ def api_parameter_from_link(link):
         return None
 
 
+def clean_html(raw_message):
+    """
+    Removes some HTML tags from input and replaces it with markdown
+    """
+    raw_message = regex.subf("<i>(.*)</i>", "*{1}*", raw_message)
```

Comment:
`(.*)` is greedy; it'll match the closing tag. You want `([^<]+)`.

Reply:
`(.*?)` should also work.

Reply:
@NobodyNada The goal is to convert HTML formatted watches and blacklists to markdown instantly instead of manually fixing it after running the command.

This specifically applies to regexes like the one linked, and one that came before it, where the raw text contains chars that are interpreted as formatting instead of keeping the plain format. 

Examples with the appropriate form and the HTML form:

| Markdown (the goal) | HTML |
| --- | --- |
|  <code>viet\W\*cruise\W\*(?:travel\W\*agency\|tours)</code> | `viet\W<i>cruise\W*(?:travel\W</i>agency\|tours)` |
| `auto\W*link\W*pk` | `auto\W<i>link\W</i>pk` |
| `total\W*credit\W*restoration` |`total\W<i>credit\W</i>restoration` |

Reply:
@LunarWatcher I was asking specifically about the regex101 link -- is it a regex meant to *match* HTML tags, or are the HTML tags supposed to be cleaned out?

Reply:
never mind, I just realized that regex is an example of the regex Art suggested.  My mistake; sorry for the noise.